hathor/verification/verification_service.py

```diff
@@ -28,11 +29,18 @@
 
 
 class VerificationService:
-    __slots__ = ('_settings', 'verifiers')
+    __slots__ = ('_settings', 'verifiers', '_tx_storage')
 
-    def __init__(self, *, settings: HathorSettings, verifiers: VertexVerifiers) -> None:
+    def __init__(
+        self,
+        *,
+        settings: HathorSettings,
+        verifiers: VertexVerifiers,
+        tx_storage: TransactionStorage | None = None,
+    ) -> None:
         self._settings = settings
         self.verifiers = verifiers
+        self._tx_storage = tx_storage
```

Comment:
Indeed, I mentioned this in the Acceptance Criteria. I'll use it in the next PR, but I added it in this one just to keep the next one smaller, since this one lays structures for it and some changes are necessary (like updating the builders).

Reply:
Ok. Thanks!